src/orientation.cpp

```diff
@@ -42,6 +42,60 @@ void updateAdj(Environment& env, int x, int y, double y2x, double x2y) {
     env.edges(y, x).status = 2;
 }
 
+//-----------------------------------------------------------------------------
+// completeOrientationUsingConsequence
+//-----------------------------------------------------------------------------
+// Description: Complete the orientations using the consequence information
+// for edges that were not previously oriented
+//
+// Detail: completeOrientationUsingConsequence will look in the list of connected
+// edges the ones that have not been oriented using the unshielded triples and
+// have at least one variable tagged as consequence.
+// Edges matching these criteria will be oriented using consequence information:
+// from the non consequence variable to the consequence variable
+//
+// Params:
+// - Environment&: the environment structure
+// - std::vector<Triple>& : list of unshielded triples
+//--------------------------------------------------------------------------------
+void completeOrientationUsingConsequence (Environment& environment,
```

Comment:
I'm just wondering if the best place to define this function is here, bcs I suppose it exists its counterpart "completeOrientations" or "completeOrientationsUsingCotnextual" somewhere ?

Reply:
There is no counterpart "completeOrientations"  or "completeOrientationsUsingContextual".
If we intend to add these kind of counterparts, I guess it would be next to where completeOrientationsUsingConsequence has been added.